### DISCUSSION TITLE
[Support] Blog中3.16.0提及mwtsc 工具切换回 tsc 自带的监听，我需要做什么工作吗

### DISCUSSION THREAD

**Discussion by minefujiko9293** [midwayjs/midway#3934] [Q&A]
### Describe the problem（描述问题）

如题，我是不是只需要升级mwtsc包到最新版本就行？
package.json中的scripts需要修改吗？
例如 
"dev:local": "cross-env NODE_ENV=local mwtsc --watch --run @midwayjs/mock/app.js",
"build": "mwtsc --cleanOutDir"
等等，是否继续沿用mwtsc就行？

### Midway Versions（Midway 版本）

>> Start to check your midway component version...

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(not installed)
✓ @midwayjs/axios(not installed) 
✓ @midwayjs/bootstrap(3.16.2)    
✓ @midwayjs/bull(3.16.4)
✓ @midwayjs/bull-board(3.16.4)
✓ @midwayjs/cache-manager(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)✓ @midwayjs/core(3.16.2)
✓ @midwayjs/cos(not installed)   
✓ @midwayjs/cron(not installed)  
✓ @midwayjs/cross-domain(3.16.4) 
✓ @midwayjs/decorator(3.16.2)    
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)  
✓ @midwayjs/grpc(not installed)  
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(not installed)  
✓ @midwayjs/info(3.16.4)
✓ @midwayjs/jwt(3.16.4)
✓ @midwayjs/kafka(not installed) 
✓ @midwayjs/leoric(not installed)✓ @midwayjs/mikro(not installed) 
✓ @midwayjs/mock(3.16.4)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/mqtt(not installed)  
✓ @midwayjs/oss(not installed)   
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(3.16.4)        
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(not installed)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(not installed)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)  
✓ @midwayjs/tenant(not installed)✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✓ @midwayjs/upload(3.16.4)       
✓ @midwayjs/validate(3.16.4)     
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)  
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.16.4)
✓ @midwayjs/ws(not installed)
**********************************************************************
>> Check complete, all versions are healthy.
**********************************************************************

**Comment by czy88840616** [comment]
需要调整 dev/build 命令，参考 https://github.com/midwayjs/midway-boilerplate/blob/master/v3/midway-framework-koa/boilerplate/_package.json#L30
